libs/components/lists/src/lib/modules/repeater/repeater-item.component.ts

```diff
@@ -47,6 +49,43 @@ let nextContentId = 0;
 export class SkyRepeaterItemComponent
   implements OnDestroy, OnInit, AfterViewInit
 {
+  /**
+   * The first, non-disabled item is tab focusable.
+   */
+  @HostBinding()
+  public get tabindex(): 0 | -1 {
+    return this.repeaterService.items.filter((item) => !item.disabled)[0] ===
```

Comment:
So only the first, non-disabled item should be focusable? The other items are not allowed to be focusable? Just trying to understand the a11y rules behind this. If there's any information about this requirement on W3C (or somewhere similar) we should put a link in this property's JSDocs comment.

Reply:
Added comments.